datasets/tr/fcib/crawler.py

```diff
@@ -14,26 +14,23 @@
 
 # original link for assertion
 DOCX_LINK = "https://ms.hmb.gov.tr/uploads/sites/2/2024/08/A-BIRLESMIS-MILLETLER-GUVENLIK-KONSEYI-KARARINA-ISTINADEN-MALVARLIKLARI-DONDURULANLAR-6415-SAYILI-KANUN-5.-MADDEw.docx"
-XLSX_LINKS = [
-    (
-        "https://ms.hmb.gov.tr/uploads/sites/2/2024/05/B-YABANCI-ULKE-TALEPLERINE-ISTINADEN-MALVARLIKLARI-DONDURULANLAR-6415-SAYILI-KANUN-6.-MADDE.xlsx",
-        "Asset freezes pursuant to Article 6 of Law No. 6415, targeting individuals"
-        " and entities based on requests made by foreign governments.",
-        "B - Foreign government requests",
+
+# Mapping of letter prefix to full label (program) and short label
+LABEL_MAPPING = {
+    "6MADDE_ING": (
+        "Asset freezes pursuant to Article 6 of Law No. 6415, targeting individuals and entities based on requests made by foreign governments.",
+        "Foreign government requests",
```

Comment:
I think their numbering (letters) is useful for quick reference

```suggestion
        "B - Foreign government requests",
```

Reply:
for the others too